development/image-detector/main.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/kyma-project/test-infra/development/image-detector/internal/prowjob"
+	"github.com/kyma-project/test-infra/development/image-detector/internal/terraform"
+	"github.com/kyma-project/test-infra/development/pkg/securityconfig"
+	"k8s.io/test-infra/prow/config"
+)
+
+type options struct {
+	ProwConfig       string
+	JobsConfigDir    string
+	TerraformDir     string
+	SecScannerConfig string
+	KubernetesFiles  string
+}
+
+func (o *options) loadOptions(flagSet *flag.FlagSet) *flag.FlagSet {
+	flagSet.StringVar(&o.ProwConfig, "prow-config", "", "path to the prow config file")
+	flagSet.StringVar(&o.JobsConfigDir, "job-config-dir", "", "path to directory containing prow jobs definition")
+	flagSet.StringVar(&o.TerraformDir, "terraform-dir", "", "path to directory containing terraform files")
+	flagSet.StringVar(&o.SecScannerConfig, "sec-scanner-config", "", "path to security config file")
+	flagSet.StringVar(&o.KubernetesFiles, "kubernetes-dir", "", "path to kubernetes deployments")
+	return flagSet
+}
+
+func (o *options) validateOptions() error {
```

Comment:
I think we agreed to use cobra framework as an cli interface. If I'm not wrong please use it for cli flags parsing.